.github/workflows/deploy.yml

```
@@ -37,6 +37,23 @@ jobs:
       - run: yarn install
       - run: yarn build
 
+      - name: Install Capacitor Dependencies
+        run: npm install @capacitor/core @capacitor/cli @capacitor/android
+
+      - name: Build and Sign Android APK
+        uses: action-capacitor-android@v1
```

actionlint (1.7.8)

specifying action "action-capacitor-android@v1" in invalid format because owner is missing. available formats are "{owner}/{repo}@{ref}" or "{owner}/{repo}/{path}@{ref}"

(action)